app/templates/_gulpfile.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var gulp = require('gulp');
+var gutil = require('gulp-util');
```

Comment:
And to be precise, it's not a plugin :) It's a util library which appends to be prefixed by 'gulp-' :)